{"version":1,"defects":{"Tests\\Unit\\RouterTest::test_checking_if_it_register_a_get_route":3,"Tests\\Unit\\RouterTest::test_checking_if_register_a_post_route_works":3,"Tests\\Unit\\UserControllerTest::test_that_registering_user_works":5,"Tests\\Unit\\UserControllerTest::test_that_login_works":5,"Tests\\Unit\\UserControllerTest::test_that_logout_works":5,"Tests\\Unit\\UserControllerTest::test_that_if_normal_user_is_able_to_go_to_admin_page":5,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_login_to_admin_page":5,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_add_user":5,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_delete_user":5,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_update_user":5,"Tests\\Unit\\RouterTest::test_that_there_are_no_routes_when_router_is_created":4,"Tests\\Unit\\RouterTest::test_to_check_if_registering_routes_work":4,"Tests\\Unit\\ExceptionsTest::it_throws_route_not_found_exception with data set #2":3,"Tests\\Unit\\ExceptionsTest::it_throws_route_not_found_exception with data set #3":3,"Error":4,"Tests\\Unit\\RouterTest::test_that_it_resolves_route_from_a_closure":4},"times":{"Tests\\Unit\\RouterTest::test_checking_if_registering_routes_work":0.001,"Tests\\Unit\\RouterTest::test_checking_if_it_register_a_get_route":0,"Tests\\Unit\\RouterTest::test_checking_if_register_a_post_route_works":0,"Tests\\Unit\\RouterTest::test_to_check_if_registering_routes_work":0,"Tests\\Unit\\RouterTest::test_to_check_if_it_register_a_get_route":0,"Tests\\Unit\\UserControllerTest::test_that_registering_user_works":0,"Tests\\Unit\\UserControllerTest::test_that_login_works":0,"Tests\\Unit\\UserControllerTest::test_that_logout_works":0,"Tests\\Unit\\UserControllerTest::test_that_if_normal_user_is_able_to_go_to_admin_page":0,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_login_to_admin_page":0,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_add_user":0,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_delete_user":0,"Tests\\Unit\\UserControllerTest::test_that_if_admin_is_able_to_update_user":0,"Tests\\Unit\\RouterTest::test_that_there_are_no_routes_when_router_is_created":0,"Tests\\Unit\\ExceptionsTest::it_throws_route_not_found_exception with data set #0":0.001,"Tests\\Unit\\ExceptionsTest::it_throws_route_not_found_exception with data set #1":0,"Tests\\Unit\\ExceptionsTest::it_throws_route_not_found_exception with data set #2":0,"Tests\\Unit\\ExceptionsTest::it_throws_route_not_found_exception with data set #3":0,"Error":0.001,"Tests\\Unit\\RouterTest::test_that_it_resolves_route_from_a_closure":0,"Tests\\Unit\\RouterTest::test_that_it_resolves_route":0}}